kitsune/wiki/urls.py

```diff
@@ -8,52 +8,52 @@
 # These patterns inherit (?P<document_slug>[^\/]).
 document_patterns = [
     re_path(r"^$", views.document, name="wiki.document"),
```

Comment:
You'll have to move this one out of the `document_patterns` and into the `urlpatterns` in order to avoid forcing all document URL's to include a trailing `/`.